doc/developer/mzbuild.md

```diff
@@ -362,6 +362,14 @@ publish: true
      Cargo is always invoked with the `--release` flag. The binary will be
      stripped of debug information unless `strip: false` is requested.
 
+     In rare cases, it may be necessary to extract files from the build
```

Comment:
This feels akin to a volume or a COPY instruction, both of which let you specify the source file's destination; given that that isn't an option, I'd recommend a note that lets the reader know where this moves the extracted files.

Reply:
👍 done.